### PR TITLE
Update ConnectButton visuals

### PR DESCRIPTION
**Pull request by lzanita09**
* Copy changes
* Button size tweaks
* Simplify TextView padding adjustment logic
* Always draw ConnectButton border

### CODE REVIEW

File: connect-button/src/main/java/com/ifttt/connect/ui/StartIconDrawable.java [hunk 1]
```diff
@@ -55,9 +54,6 @@
         this.iconSize = iconSize;
         this.initialBackgroundSize = initialBackgroundSize;
         this.onDarkBackground = onDarkBackground;
-        this.startIconBackgroundColor =
-                onDarkBackground ? ContextCompat.getColor(context, R.color.ifttt_start_icon_background_on_dark)
-                        : Color.BLACK;
```
Comment (priyaashah) [inline connect-button/src/main/java/com/ifttt/connect/ui/StartIconDrawable.java]: is this intentional? This color is used below, if we don't set it in the constructor, how can we use it where required?

**Comment by priyaashah** [comment]
I am assuming all the dimens are same as those for the connect button in the app (https://github.com/IFTTT/IFTTTAndroid/pull/3924)

**Comment by lzanita09** [comment]
Yep.

File: connect-button/src/main/java/com/ifttt/connect/ui/StartIconDrawable.java (hunk 1 above)
Comment (lzanita09) [reply connect-button/src/main/java/com/ifttt/connect/ui/StartIconDrawable.java]: We no longer use this field because we are going to use the `backgroundColor`, which will be set later on in the process, to derive the background color for the start icon.

File: connect-button/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <dimen name="ifttt_text_padding_horizontal">72dp</dimen>
+    <dimen name="ifttt_text_padding_horizontal">@dimen/ifttt_connect_button_height</dimen>
```
Comment (priyaashah) [inline connect-button/src/main/res/values/dimens.xml]: I would say we could leave this as `72dp` instead of setting it as the connect button height. Those two dimens are unrelated and if we decide to change height, this will change too

Comment (lzanita09) [reply connect-button/src/main/res/values/dimens.xml]: They are related actually, the padding should match exactly how tall the connect button is.
